modules/sandboxed-containers-fips-validation.adoc

```diff
@@ -0,0 +1,13 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+[id="sandboxed-containers-fips-validation_{context}"]
+
+= FIPS validation for {sandboxed-containers-first}
+
+The {sandboxed-containers-operator} is based on the {op-system-first} extensions concept. The sandboxed containers {op-system} extension contains RPMs for Kata, QEMU, and its dependencies. You can enable them by using the `MachineConfig` resources that the Machine Config Operator provides.
```

Comment:
```suggestion
The {sandboxed-containers-operator} is based on the {op-system-first} extensions concept. The {sandboxed-containers-first} {op-system} extension contains RPMs for Kata, QEMU, and its dependencies. You can enable them by using the `MachineConfig` resources that the Machine Config Operator provides.
```

Reply:
My bad - this was actually a draft module I started writing at first before I realized a similar module already existed and I used it instead... The content here is for RHCOS. Glad you caught it - I deleted it and committed the changes. :)